APIManager.js

```diff
@@ -0,0 +1,85 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+    constructor() {
+        this.data = {}
+    }
+    loadUser() {
+        const result = $.ajax({
+            method: 'get',
+            url: `https://randomuser.me/api/`,
+            async: false
+        });
```

Comment:
where is the success and error part of the `.ajax` call? it is important to get used to error handling